### PR TITLE
Document (Omni/Spot)Light ignoring Spatial's scale property (3.x)

### PR DESCRIPTION
**Pull request by Calinou**
`3.x` version of https://github.com/godotengine/godot/pull/67098.

This closes https://github.com/godotengine/godot/issues/67058.

### CODE REVIEW

File: doc/classes/Light.xml [hunk 1]
```diff
@@ -51,6 +51,7 @@
 		</member>
 		<member name="light_size" type="float" setter="set_param" getter="get_param" default="0.0">
 			The size of the light in Godot units. Only considered in baked lightmaps and only if [member light_bake_mode] is set to [constant BAKE_ALL]. Increasing this value will make the shadows appear blurrier. This can be used to simulate area lights to an extent.
+			[b]Note:[/b] [member light_size] is not affected by [member Spatial.scale] (on the light itself or its parents).
```
Comment (skyace65) [inline doc/classes/Light.xml:54]:
```suggestion
			[b]Note:[/b] [member light_size] is not affected by [member Spatial.scale] (The light's scale or its parent's scale).
```
I had to read that 4 times to figure out what it was saying, I think this should be more clear.

**Review by skyace65 — APPROVED** [review]
I think the part in the parenthesis could be more clear, but I'll defer to other peoples opinion on it. Everything else looks good so I'll approve this.

Edit: Should there be a note added to the spatial node like in the master version of this PR?

**Comment by Calinou** [comment]
> Edit: Should there be a note added to the spatial node like in the master version of this PR?

Added.

**Review by clayjohn — APPROVED** [review]

**Comment by akien-mga** [comment]
Needs rebase to fix iOS build issue (since fixed in `3.x`).

**Comment by akien-mga** [comment]
> Needs rebase to fix iOS build issue (since fixed in `3.x`).

Bump.

**Comment by akien-mga** [comment]
Thanks!

**Comment by timothyqiu** [comment]
Cherry-picked for 3.5.2

**Pull request merged**